### PR TITLE
Add test for extended models

### PR DESCRIPTION
**Pull request by yakky**
This just adds tests for uploading extended objects

### CODE REVIEW

File: filer/test_utils/extended_app/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('filer', '0004_auto_20160328_1434'),
```
Comment (mkoistinen) [inline filer/test_utils/extended_app/migrations/0001_initial.py]: Please use 0001_initial, if possible here.

**Comment by yakky** [comment]
@mkoistinen any more fixes needed to merge this?

**Comment by czpython** [comment]
👍

**Comment by mkoistinen** [comment]
LGTM, but please add a changelog entry.

**Comment by yakky** [comment]
@mkoistinen changelog added.
Merging

**Pull request merged**